src/interoperability.md

```diff
@@ -1 +1,17 @@
-Contents will be added as part of https://github.com/sora-xor/sora-docs/pull/44
+# Interoperability
+
+This article is about the HASHI bridge - a decentralized two-way bridge connecting SORA and other blockchains (such as Ethereum and Polkadot). HASHI means bridge in Japanese, and it is the plumbing that links many different chains, like pipes connecting the world's assets. The initial version of this bridge was a significant component of the plans for the beta launch of the SORA network in Q1 2021.
+
+## History
+
+The SORA blockchain serves as an ecosystem for robust DeFi projects like SORA's decentralized global economic system and [Polkaswap](./polkaswap), a DEX specifically designed for an interoperable future. In line with this vision, we recognize that blockchain maximalism is no longer viable, and we acknowledge the importance of secure, fast, and efficient asset migration between blockchains.
+
+As an initial step towards achieving this, we have developed the source code for [Ethereum Bridge](./hashi). In addition to Ethereum, SORA aims to establish connections with various ecosystems. Being built on Substrate, it is advantageous to utilize the built-in cross-chain transfer capabilities to facilitate communication with projects running on the Polkadot and Kusama ecosystems. To achieve this, the [Substrate bridge](./substrate-bridge) was developed.
+
+With multiple bridges already in operation, it becomes necessary to streamline the bridge interface to ensure consistent software components for cross-chain transfers across different heterogeneous blockchains. As a solution, the "Federated Bridge" was created to serve as the gateway between the SORA mainnet and the external world. The term "federated" is used because this bridge inherits certain features from a federated architecture.
```

Comment:
This is wrong. The bridge interface to unify multiple bridges to multiple chains is the HASHI bridge. 
The federated architecture of the Substrate bridge is a technical / architectural choice. This is not the logical consequence of the choice to implement a bridge interface.

```suggestion
With multiple bridges already in operation, it becomes necessary to streamline the bridge interface to ensure consistent software components for cross-chain transfers across different heterogeneous blockchains. 
As a solution, the HASHI bridge is aggregating several bridge technologies to serve as the gateway between the SORA mainnet and the external world. 

The HASHI bridge currently comprises two bridges: the Federated EVM Bridge, and the Federated Substrate Bridge

The term "federated" is used because this bridge inherits certain features from a federated architecture.
```

Would be nice to elaborate a bit on what it means for the bridge to be federated vs. trusted vs. trustless.